devenv/stacks/docker/Config.toml

```diff
@@ -0,0 +1,35 @@
+[node]
+working_dir = "/root/stacks-node/data"
+rpc_bind = "0.0.0.0:20443"
+p2p_bind = "0.0.0.0:20444"
```

Comment:
I see a few warnings like this. Maybe we can stop these to increase the performance?
`WARN [1694100874.608488] [src/net/p2p.rs:2484] [ThreadId(24)] local.80000000://(bind=0.0.0.0:20444)(pub=None): failed to learn public IP: NoSuchNeighbor
`